app/src/pages/inside/common/infoLine/description/descriptionTooltip/descriptionTooltip.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { FormattedMessage } from 'react-intl';
+import { ScrollWrapper } from 'components/main/scrollWrapper';
+import { MarkdownViewer } from 'components/main/markdown';
+import styles from './descriptionTooltip.scss';
+
+const cx = classNames.bind(styles);
+
+export const DescriptionTooltip = ({ description }) => (
+  <div>
+    {description ? (
+      <div className={cx('description-tooltip')}>
+        <ScrollWrapper>
+          <MarkdownViewer value={description} />
+        </ScrollWrapper>
+      </div>
+    ) : (
+      <FormattedMessage id="DescriptionTooltip.message" defaultMessage="No description" />
```

Comment:
It's better to hide the icon to to have more consistent behavior with grid items